functions.sh.in

```diff
@@ -88,7 +88,8 @@ readconfig() {
 	else
 		if [ -n "$gitbase" ]; then
 			case $(git config remote.origin.url) in
-			*/aports) APORTSDIR=$gitbase ;;
+			# '.git' for SSH URLs, and no suffix for HTTPS URLs
+			*/aports | */aports.git) APORTSDIR=$gitbase ;;
```

Comment:
```suggestion
			*/aports|*/aports.git) APORTSDIR=$gitbase ;;
```

I'm not sure if the spaces are going to affect the match or not, but better to just not have them.